apps/opik-backend/src/main/java/com/comet/opik/api/Span.java

```diff
@@ -39,14 +39,17 @@ public record Span(
         @JsonView({Span.View.Public.class, Span.View.Write.class}) JsonNode input,
         @JsonView({Span.View.Public.class, Span.View.Write.class}) JsonNode output,
         @JsonView({Span.View.Public.class, Span.View.Write.class}) JsonNode metadata,
+        @JsonView({Span.View.Public.class, Span.View.Write.class}) String model,
+        @JsonView({Span.View.Public.class, Span.View.Write.class}) String provider,
         @JsonView({Span.View.Public.class, Span.View.Write.class}) Set<String> tags,
         @JsonView({Span.View.Public.class, Span.View.Write.class}) Map<String, Integer> usage,
         @JsonView({Span.View.Public.class}) @Schema(accessMode = Schema.AccessMode.READ_ONLY) Instant createdAt,
         @JsonView({Span.View.Public.class}) @Schema(accessMode = Schema.AccessMode.READ_ONLY) Instant lastUpdatedAt,
         @JsonView({Span.View.Public.class}) @Schema(accessMode = Schema.AccessMode.READ_ONLY) String createdBy,
         @JsonView({Span.View.Public.class}) @Schema(accessMode = Schema.AccessMode.READ_ONLY) String lastUpdatedBy,
         @JsonView({
-                Span.View.Public.class}) @Schema(accessMode = Schema.AccessMode.READ_ONLY) List<FeedbackScore> feedbackScores){
+                Span.View.Public.class}) @Schema(accessMode = Schema.AccessMode.READ_ONLY) List<FeedbackScore> feedbackScores,
+        @JsonView({Span.View.Public.class}) @Schema(accessMode = Schema.AccessMode.READ_ONLY) Double totalEstimatedCost){
```

Comment:
For monetary calculations, you shouldn't use floating types. You should go with BigDecimal.

Reply:
Floating types don't ensure precision and rounding isn't controlled during operations, so they aren't a good fit for monetary values.

This is an estimation and we don't charge users, so it isn't that bad, but we should aim to do the right thing.